kernel/kernel-config-4.9.x

```diff
@@ -2206,6 +2231,7 @@ CONFIG_SECURITY=y
 # CONFIG_SECURITYFS is not set
 # CONFIG_SECURITY_NETWORK is not set
 # CONFIG_SECURITY_PATH is not set
+# CONFIG_INTEL_TXT is not set
```

Comment:
curious why in [R38](https://github.com/clearcontainers/packaging/pull/171/files#diff-e11b524d9b4bb32562113b63b912ccbbR38) is enabled and here is disabled

Reply:
This option is the feature, R38 is the boolean flag to show this option